_docs/geyser/standalone.md

```diff
@@ -0,0 +1,78 @@
+
+### Standalone Setup (Geyser and Floodgate are not on the same server)
+If you want to offload Geyser's processing you can host Geyser (Standalone) somewhere else and make it redirect to your server.
+**Warning:** Do not forget to edit the remote section of your Geyser config.
+See the [Floodgate](/floodgate/) wiki page for more details.
+
+## Port Forwarding
+
+Unlike Minecraft Java Edition, Bedrock Edition runs on port 19132 on the UDP protocol. When port forwarding, make sure to allocate to 19132 UDP or another UDP port. For many server hosting providers, you will simply need to change your Bedrock listening port (see [here](/geyser/supported-hosting-providers/) for a list of supported providers).
+
+## Termux (Android)
+Please read the disclaimer [here](/geyser/setup/#standalone-setup) before continuing.
+1. Download and install [Termux](https://termux.com/)
+2. Run `pkg install openjdk-17`
+3. Run `wget https://ci.opencollab.dev/job/GeyserMC/job/Geyser/job/master/lastSuccessfulBuild/artifact/bootstrap/standalone/build/libs/Geyser-Standalone.jar`
+4. Run `java -jar Geyser-Standalone.jar`
+
+OR
+
+We have an automated setup script for clean Termux installs, which might not work for all users. If the manual guide above does not work, try this.
+Run this to start the download/install:
+```
+curl https://gist.githubusercontent.com/rtm516/e3e07d6595ee41e05a38b03c0f4d7a80/raw/install.sh | bash
+```
+
+## NewTerm 2 (iOS)
+**Note:** A jailbreak is required. You can find what jailbreak to use for your device [here](https://docs.google.com/spreadsheets/d/11DABHIIqwYQKj1L83AK9ywk_hYMjEkcaxpIg6phbTf0/edit?usp=sharing).
+1. Install [Filza File Manager](http://cydia.saurik.com/package/com.tigisoftware.filza/).
+2. Install [NewTerm 2](https://chariz.com/get/newterm).
+4. Download jre-16 for iOS and install it using Filza [here](https://github.com/PojavLauncherTeam/PojavLauncher_iOS/releases/download/v16-openjdk/openjdk-16-jre_16.0.0+git20201217.8383f41-2_iphoneos-arm.deb).
+3. Download my modified java commands, and install it using Filza [here](https://cdn.discordapp.com/attachments/558829512633090048/834014323755319306/com.letschill.java_0.1_iphoneos-arm.deb).
+4. Open NewTerm 2 and run `wget https://ci.opencollab.dev/job/GeyserMC/job/Geyser/job/master/lastSuccessfulBuild/artifact/bootstrap/standalone/build/libs/Geyser-Standalone.jar`
+5. Run `java -jar Geyser-Standalone.jar`.
+6. You can find Geyser server running in MCPE!
+
+**Note:**
+Due to iOS's environment, if your device has low specs, iOS might end up killing NewTerm 2 while you're playing, stopping the server. There is no fix for that because of how iOS works.
+
+You may encounter some issues. If you do, run `su` then enter the root password (default is `alpine`) for root access. Then, run the server like you normally would, and it should work just fine.
```

Comment:
This is probably an awful idea, security wise. Should we remove this section, or at least add a disclaimer?

Reply:
No idea honestly - I'd remove it tbh